scalalib/api/src/mill/scalalib/api/ZincWorkerUtil.scala

```diff
@@ -65,6 +65,10 @@ trait ZincWorkerUtil {
     case _ => scalaVersion
   }
 
+  private def binaryVersion(version: String): ( /*major: */ Int, /*minor:*/ Int) = version match {
```

Comment:
The name `binaryVersion` is a bit misleading, since in Scala 3, a binary version has only the major position.

Reply:
I've renamed it to `partialVersion` so that it would be more explicit, would not clash with `xBinaryVersion` methods defined in other places, and would give a hint about. 
In the 1.x it was named `minorMajorVersion` which unfortunately was confusing when we get rid of named tuples